CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the "vscode-debug-adapter-apache-camel" extension will be
 ## 0.8.0
 
 - Provide settings for specifying the Camel version to run integrations with Camel JBang CLI
+- Provide settings for specifying the Red Hat Maven Repository
```

Comment:
the most interesting part of the feature is that the repository is automatically added when the camel version contains redhat